articles/azure-monitor/app/connection-strings.md

```diff
@@ -13,10 +13,14 @@ ms.reviewer: cogoodson
 Connection strings specify to which Application Insights resource your instrumented application should send telemetry data. A connection string is composed of multiple settings, each represented as a key-value pair and separated by semicolons. This consolidated configuration method simplifies the setup process by eliminating the need for multiple proxy settings.
 
 > [!IMPORTANT]
-> The connection string contains an ikey, which is a unique identifier used by the ingestion service to associate telemetry to a specific Application Insights resource. ***Ikey unique identifiers aren't security tokens or security keys, and aren't considered secrets.***
->
+> The connection string contains an ikey and an ApplicationId.
+> 
+>The ikey is a unique identifier used by the ingestion service to associate telemetry to a specific Application Insights resource. ***Ikey unique identifiers aren't security tokens or security keys, and aren't considered secrets.***
+>Connection Strings are enhanced by adding the ApplicationId value. This update is a new feature that supports auto-instrumentation for scenarios utilizing the Open Telemetry SDK.
+
```

Comment:
```suggestion
>Connection strings are enhanced by adding the ApplicationId value. This update is a new feature that supports automatic instrumentation for scenarios that utilize the Open Telemetry SDK.
```
Fixing capitalization and removing line space and making minor edits to align with Microsoft style.